example/src/app/app.component.html

```diff
@@ -0,0 +1,13 @@
+<h1>Jane Doe</h1>
+<div class="job-title">Web Developer</div>
```

Comment:
Čia visai gerai, kad klasę panaudojai, bet logiškai šis element'as nėra tik `<div>`. Sakyčiau, kad čia tinkamiau būtų buvę `<h3>` panaudoti